biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -243,17 +250,17 @@ real(r8) :: tcwood             ! Temperature response function for wood
     ! Ball-Berry minimum leaf conductance, unstressed (umol H2O/m**2/s)
     ! For C3 and C4 plants
     ! -----------------------------------------------------------------------------------
-    real(r8), dimension(0:1) :: bbbopt 
+ 
 
     associate(  &
          c3psn     => EDPftvarcon_inst%c3psn  , &
          slatop    => EDPftvarcon_inst%slatop , & ! specific leaf area at top of canopy, 
                                                   ! projected area basis [m^2/gC]
-         woody     => EDPftvarcon_inst%woody)     ! Is vegetation woody or not? 
+         woody     => EDPftvarcon_inst%woody  , & ! Is vegetation woody or not?
+         stomatal_intercept   => EDPftvarcon_inst%stomatal_intercept ) !stomatal intercept for Ball-Berry model and Medlyn model
```

Comment:
@Qianyuxuan shall we just comment as:

"Intercept or g0 of stomatal conductance model" ?

Reply:
@serbinsh okay.